arwen/elrondapi/generate/eiGenWriteRustVMHooksTrait.go

```diff
@@ -12,7 +12,12 @@ func WriteRustVMHooksTrait(out *os.File, eiMetadata *EIMetadata) {
 // !!!!!!!!!!!!!!!!!!!!!! AUTO-GENERATED FILE !!!!!!!!!!!!!!!!!!!!!!
 // !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
-pub trait VMHooks: 'static {
+use std::ffi::c_void;
+
+#[rustfmt::skip]
+pub trait VMHooks: core::fmt::Debug + 'static {
+    fn set_context_ptr(&mut self, context_ptr: *mut c_void);
```

Comment:
Consider renaming `set_context_ptr` to `set_vmhooks_pointer`, to avoid using the term "context" too much.